src/sentry/release_health/metrics.py

```diff
@@ -607,7 +607,7 @@ def f(row: Mapping[str, Union[int, str]]) -> ProjectOrRelease:
             select=query_cols,
             where=where_clause,
             groupby=group_by_clause,
-            granularity=Granularity(LEGACY_SESSIONS_DEFAULT_ROLLUP),
+            granularity=Granularity(24 * 60 * 60),  # daily
```

Comment:
Daily aggregation should be good enough for a 90 day period.

Reply:
won't this introduce more differences? we aligned those rollups because different rollups created diffs

Reply:
Sorry I missed this comment before merging. In theory, yes, but this function has not reported any deltas in the last 90 days even though the queried time range was completely different between sessions and metrics (3 days vs. 90 days). I don't know why.